docs/community/frequently_asked_questions.md

```diff
@@ -0,0 +1,77 @@
+# Frequently Asked Questions (FAQ)
+
+---
+
+##### [Large Language Models (LLM)](./faq/llms.md)
+
+Discover how to tailor LLMs, explore available models, understand cost implications, and switch between languages.
+
+```{toctree}
+---
+maxdepth: 1
+hidden: true
+---
+faq/llms.md
+
+```
+
+---
+
+##### [Embeddings](./faq/embeddings.md)
+
+How to customize the embedding, Which embeddings model choose, their pros and cons
+
+```{toctree}
+---
+maxdepth: 1
+hidden: true
+---
+faq/embeddings.md
+
+```
+
+---
+
+##### [Vector Database](./faq/vector_database.md)
+
+Get insights on personalizing vector databases, delve into database options
+
+```{toctree}
+---
+maxdepth: 1
+hidden: true
+---
+faq/vector_database.md
+
+```
+
+---
+
+##### [Query Engines](./faq/query_engines.md)
+
+Know more about query engines and their possibilities
+
+```{toctree}
+---
+maxdepth: 1
```

Comment:
please add `hidden: true` underneath this as with the other sections. Otherwise, I believe it renders a redundant bulleted item for `Query Engines`